site/content/blog/2017-08-22-zap-browser-launch/index.md

```diff
@@ -22,7 +22,7 @@ You will now get a new 'Launch Browser' option in the Quick Start tab:
   
 {{< img "images/launch-browser.png" >}}  
 To see a demo of this feature see the following video:
-{{< youtube lYHwR7UEggA >}}
+{{< youtube uuid="lYHwR7UEggA" >}}
```

Comment:
This can be reverted, being fixed in the other pull request.

Reply:
done